### PR TITLE
Default language for content proxy

### PR DESCRIPTION
**Pull request by dtaylor7**
Ddefault language when getting keys from the content proxy

### CODE REVIEW

File: src/middleware/resolveTemplate.js [hunk 1]
```diff
@@ -11,7 +11,9 @@ const viewTargets = (step, viewFolders) => {
   return viewFolders
     .map(folder => [
       path.resolve(path.join(folder, `${step.name}.html`)),
-      path.resolve(path.join(folder, `${step.name}.template.html`))
+      path.resolve(path.join(folder, `${step.name}.template.html`)),
+      path.resolve(path.join(folder, `${step.name.toLowerCase()}.html`)),
+      path.resolve(path.join(folder, `${step.name.toLowerCase()}.template.html`))
```
Comment (endamccormack) [inline src/middleware/resolveTemplate.js]: This seems very specific to your implementation.

Comment (dtaylor7) [reply src/middleware/resolveTemplate.js]: i did try changing files to use uppercase, but when i logged them out on CNP it couldnt find them....

**Comment by michaeldfallen** [comment]
Defaulting language should already be done by i18Next?

**Comment by michaeldfallen** [comment]
https://github.com/hmcts/one-per-page/blob/7b3098ec0983996dde3baf914f1ce915f7d4ec4d/src/i18n/i18Next.js#L7
☝️ if a key doesn't exist it should fallback to `en`.

**Comment by dtaylor7** [comment]
@michaeldfallen that default doesnt seem to work for some reason.

**Comment by dtaylor7** [comment]
have used the default lang instead

**Pull request merged**